Special Building Changes

james9017 opened the pull request:
(Just committing for a general backup, you can still review changes/additions if you like though)
- Added 'btwk' prefixes to our added buildings.
- Added new "special mines" for salt, tin, lead, copper, iron, silver and gold, and added all (but salt) to locations across the map
- Addition of a special building slot in Sanquhar via history file is incorrect. This is known and will be fixed later.
- Added new building script values to calculate the tax levels of each building. May do this for the other modifiers also, but I don't always have them increase by a fixed amount (trying to make it so players have an incentive to upgrade to later mine tiers).
- Overrides for silver and copper base tax script values don't work. Not sure why. New values & formulae work fine though for other mines active at game start (haven't checked rounding errors for my 3.d.p. values yet.)
- Buildings currently use vanilla mine format, i.e. have 4 tiers, upgrades unlocked at same innovations as vanilla, same "can_construct(_potential)" blocks. Make any suggested changes to this layout if you deem necessary.
- Need help on localisation ideas, i.e.: should each metal mine tier (salt will be different) be named the same (all have a tier called something like 'Small Mining Community' or 'On-Site Forges'...)? Or tie in their unique uses someway?
- Need to remake Cornish Tin Mines w.r.t. the new mine format. (tin mines, but better)
- May add special iron mine to the Weald. Read an article saying most of the Iron in Roman Britain was sourced from here, but that the map terrain is impassable where the mines were (based on map from DC) I think, hence add bonuses to the mines to reflect this.

- Added placeholders for Rock of Dumbarton, Merlin's Tor (may conflict with holy site grand cathedral @ Avalon already, not sure) & the baths in Bath.

james9017 committed:
Special Building Changes

(Just committing for a general backup, you can still review changes/additions if you like though)
- Added 'btwk' prefixes to our added buildings.
- Added new "special mines" for salt, tin, lead, copper, iron, silver and gold, and added all (but salt) to locations across the map
- Addition of a special building slot in Sanquhar via history file is incorrect. This is known and will be fixed later.
- Added new building script values to calculate the tax levels of each building. May do this for the other modifiers also, but I don't always have them increase by a fixed amount (trying to make it so players have an incentive to upgrade to later mine tiers).
- Overrides for silver and copper base tax script values don't work. Not sure why. New values & formulae work fine though for other mines active at game start (haven't checked rounding errors for my 3.d.p. values yet.)
- Buildings currently use vanilla mine format, i.e. have 4 tiers, upgrades unlocked at same innovations as vanilla, same "can_construct(_potential)" blocks. Make any suggested changes to this layout if you deem necessary.
- Need help on localisation ideas, i.e.: should each metal mine tier (salt will be different) be named the same (all have a tier called something like 'Small Mining Community' or 'On-Site Forges'...)? Or tie in their unique uses someway?
- Need to remake Cornish Tin Mines w.r.t. the new mine format. (tin mines, but better)
- May add special iron mine to the Weald. Read an article saying most of the Iron in Roman Britain was sourced from here, but that the map terrain is impassable where the mines were (based on map from DC) I think, hence add bonuses to the mines to reflect this.

- Added placeholders for Rock of Dumbarton, Merlin's Tor (may conflict with holy site grand cathedral @ Avalon already, not sure) & the baths in Bath.

james9017 commented:
- Also need to add Alderly Edge copper mine per Quexcy's suggestion

james9017 committed:
Special Building Changes - Draft 2

Backup push for posterity's sake.
- Added general localisation for special mine tiers
- Yet to add special mine building type localisation
- Yet to add salt field building type localisation (and may move this to special buildings)
- Removed most tier 1 special mines present at game start, and added special building slots instead
- Gave Cornish tin mines a set of upgrades up to tier 4 to match other mines. Still requires proper localisation.
- Added Glastonbury Tor (Merlin's Tor), Dumbarton Rock & Roman Baths special buildings. Localisation an effects ready for review. Need to find better icons for the buildings.
- Fixed copper & silver mine base tax values not working.
- NOTE: Forgot to test the new cultural trigger I added, but I'm going to bed now so I'll check a look tomorrow.
- TODO: Add Holy Island. Not sure best way to approach this. Was thinking to tie it in with Irish missionaries, if we decide to implement them, seeing as it was St. Aidan who established the monastery. Maybe disable religious special buildings here & have a MTTH event/decision fire for Christian rulers of Lindisfarne to invite monks?

DC123456789 commented on Britannia/common/buildings/BTWK_special_buildings.txt:
```diff
@@ -177,24 +177,122 @@ offas_dyke_01 = {
+	can_construct = {
+		culture = { 
+			has_innovation = innovation_crop_rotation 
+			culture_is_brythonic_culture_trigger = yes
```
Sort of think that this building doesn't need a cultural requirement at all

james9017 replied:
No it doesn't, I don't know why I put it there - the building exists at game start

DC123456789 commented on Britannia/common/buildings/BTWK_special_buildings.txt:
```diff
@@ -177,24 +177,122 @@ offas_dyke_01 = {
+
+	cost_gold = expensive_building_tier_3_cost
 
+	province_modifier = {
```
Modifier feels quite weak tbh (same applies for the other buildings, except maybe Aquae Sulis Baths 1), unless you intend to make them upgradable

james9017 replied:
Wasn't particularly sure of how to best balance them, so I was probably a bit conservative with my bonuses. If you have any suggestions for boosting the modifiers, I'm all ears.

DC123456789 commented on Britannia/common/buildings/BTWK_special_buildings.txt:
```diff
@@ -177,24 +177,122 @@ offas_dyke_01 = {
+	can_construct = {
+		culture = { 
+			has_innovation = innovation_crop_rotation 
+			culture_is_imperial_like_culture_trigger = yes
```
Don't think this should have a cultural requirement at all - Glastonbury Abbey reached its zenith under the Saxons/Normans

james9017 replied:
Remnant from before discussions on changes to the Tor, will amend

DC123456789 commented on Britannia/common/buildings/BTWK_special_buildings.txt:
```diff
@@ -177,24 +177,122 @@ offas_dyke_01 = {
+	province_modifier = {
+		fort_level = 2
+		defender_holding_advantage = 10
+		station
```
Unfinished line? Causing a syntax error

james9017 replied:
Unfinished line

james9017 replied:
Was going to put stationed archers toughness (i.e. fort is elevated on the rock => high ground advantage) - sound okay?   I know that they wouldn't have that "high ground" in battle elsewhere, just thought it might be an interesting modifier to throw in given the context

DC123456789 replied:
Think that's fine - alternative I guess would be whatever cultural MAA Cumbrians have (do they even have one?)

james9017 replied:
No they don't

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -141,3 +157,31 @@
+ ################## 
+ building_type_btwk_dumbarton_rock_01:0 "The Rock of Dumbarton"
+ building_type_btwk_dumbarton_rock_01_desc:0 "#italic Dumbarton Rock#! is a rocky outcrop sitting on the edge of the River Clyde overlooking the settlement of Dumbarton, providing ample defence for the fort sitting atop it. It is commonly associated with being the primary seat of the medieval Kingdom of Strathclyde and is even rumoured to have hosted the legendary wizard Merlin at one time..."
+ building_btwk_dumbarton_rock_01:0 "$building_type_btwk_glastonbury_tor_01$"
```
Wrong keys

DC123456789 commented on Britannia/common/on_action/BTWK_game_start.txt:
```diff
@@ -1772,3 +1773,31 @@ BTWK_de_jure_setup = {
 	}
 }
 
+BTWK_special_buildings_setup = {
```
Any specific reason you want to put this here instead of in the province history (like how vanilla does it)?

james9017 replied:
You can't add special building slots in the history files. This is also how it's implemented in vanilla.

DC123456789 replied:
Oh really? I saw a lot of special_building_slot = X in the vanilla files, those don't work?

james9017 replied:
It didn't seem to work when I tried to add the mines initially... like they weren't appearing at all that way. I'll have another look.

DC123456789 commented on Britannia/common/scripted_triggers/BTWK_cultural_triggers.txt:
```diff
@@ -178,4 +178,15 @@ is_imperial_like_culture_trigger = {
 	}
+}
+
+culture_is_imperial_like_culture_trigger = {
```
Shouldn't be independent of the is_imperial_like_culture_trigger 
Alternatively, do you actually need this trigger right now?

james9017 replied:
Nope, just about to remove it

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -26,6 +26,16 @@
+ building_btwk_mines_03:0 "Support Structures"
+ building_btwk_mines_03_desc:0 "Before the shaft can be expanded further, measures must be put in place to prevent its future in-collapse, such as lining our tunnels regularly with wood and stone, and ensuring that the groundwater is regularly drained out." 
+ building_btwk_mines_04:0 "Deeper Shafts"
+ building_btwk_mines_04_desc:0 "To access the minerals deeper underground, we must employ the practice of #italic shaft sinking#! - vertically (or near-vertically) deepening a mine shaft, to gain better access to deeply bedded minerals. The excavation work here was one of the most dangerous jobs in the medieval world, and as such was usually only tasked to slaves and criminals."
```
Not totally sure we want to mention the word "medieval" tbh

james9017 replied:
Wasn't sure myself but I thought I'd leave it in and see what you thought. Will have a look at modifying it

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -39,78 +49,84 @@
- building_btwk_gold_mines_03_desc:0 ""
- building_btwk_gold_mines_04:0 ""
- building_btwk_gold_mines_04_desc:0 ""
+ building_type_btwk_gold_mines_01_desc:0 "This area is rich with one of history's most coveted, precious metals - gold. While the metal is of course valuable, it was mostly used in the crafting of high value decorative pieces for those who could afford them. During this period, a shift was made to using predominantly silver as the standard for currency in Europe - a change which would not be reve ...
```
I think we don't really want to be directly referencing developments outside our time period - feels kind of out of context here

james9017 replied:
Ties into same issue with Tor localisation - wasn't sure on the frame of reference they were being described from. Change it to be exclusively period specific?

DC123456789 replied:
Yeah I think generally the convention for building descriptions would be to describe things from the PoV of the ingame characters, or at least as though the building and its functions are current

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -39,78 +49,84 @@
- building_btwk_cornish_tin_mines_01_desc:0 "$building_type_btwk_cornish_tin_mines_01_desc$"
+ building_type_btwk_cornish_tin_mines_01_desc:0 "The valleys of southwestern Britain (such as the Tamar Valley) are well known for their rich deposits of metals, particularly that of tin. The most common practice used for their extraction is a process called #italic streaming#!, which may only be performed in and around river beds."
+ building_btwk_cornish_tin_mines_01:0 "$building_btwk_mines_01$"
+ building_btwk_cornish_tin_mines_01_desc:0 "$building_type_btwk_cornish_tin_mines_01_desc$\n\nA small groups of miners has established a settlement in one such tin rich area."
```
Grammar

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -141,3 +157,31 @@
+ ################
+ # BATH'S BATHS #
+ ################
+ building_type_btwk_aquae_sulis_baths_01:0 "Roman Baths"
```
Possibly "Baths of Aquae Sulis"? Roman Baths were in every Roman city tbh

james9017 replied:
Was going to put that, but again thought we were localising using English names so avoided using it (as Baths of 'Bath' sounds well stupid lol)

DC123456789 replied:
Hmm I guess my thought was that Aquae Sulis is the standard English name for Bath in the Roman era (which I think is generally the case for Roman names of cities that still exist today)

james9017 replied:
That's fine. I've changed it to Aquae Sulis anyway as it does sound correct. I just misunderstood what you meant by using 'English' names, which has been resolved by your comment on Alt Clut/Strathclyde

DC123456789 replied:
I guess by "English name" I really mean "English historiographical name", i.e. the name you're mostly likely to see when opening a paper or textbook or encyclopedia.

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -141,3 +157,31 @@
+ building_type_btwk_aquae_sulis_baths_01:0 "Roman Baths"
+ building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans built a series of baths around some natural springs from which the local Britons worshipped their God #italic Sulis #!(hence, the name of the settlment #italic Aquae Sulis#!). Over the following centuries during Roman occupation, this settlement would grow into a large resort complex, of which its ba ...
+
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths"
```
Possibly "Decayed/Abandoned Baths"?

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -141,3 +157,31 @@
+
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nThe departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms, rather than to the upkeep of amenities like this spa. As such, the once magnificent resort has fallen into disuse and decay, and stands more as a monument to an age long past."
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths - Restored"
```
"Restored Roman Baths"?

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -141,3 +157,31 @@
+ building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans built a series of baths around some natural springs from which the local Britons worshipped their God #italic Sulis #!(hence, the name of the settlment #italic Aquae Sulis#!). Over the following centuries during Roman occupation, this settlement would grow into a large resort complex, of which its ba ...
+
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nThe departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms, rather than to the upkeep of amenities like this spa. As such, the once magnificent resort has fallen into disuse and decay, and stands more as a monument to an age long past."
```
Possibly something about the natural springs still being used?

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -141,3 +157,31 @@
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nThe departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms, rather than to the upkeep of amenities like this spa. As such, the once magnificent resort has fallen into disuse and decay, and stands more as a monument to an age long past."
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths - Restored"
+ building_btwk_aquae_sulis_baths_02_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nRenovations have been carried out on the spa resort to allow for the recreational bathing the Romans once enjoyed. This has encouraged a larger influx of freemen to bring their custom to the resort - a great boon to the local economy."
```
"recreational" feels a bit too casual given how important baths were to the Roman way of life - I would probably lean into the societal aspect of the baths a lot more
In the same vein "freemen" is also kind of a weird word to be using here, I think - "citizen" might be better if you want to keep the wording

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -141,3 +157,31 @@
+ # GLASTONBURY TOR #
+ ###################
+ building_type_btwk_glastonbury_tor_01:0 "Glastonbury Tor"
+ building_type_btwk_glastonbury_tor_01_desc:0 "The pre-Christian history of the tor is quite unknown. Archeological evidence indicates there was no permanent settlement until after around the 6th century AD, and would later give way to Glastonbury Abbey. Arthurian legend, however, suggests this as being the homestead of the mythical druid Merlin..."
```
Talking about "pre-Christian history" and "Archeological evidence" definitely sounds a bit weird here
"Arthurian legend" and "mythical druid" is also kind of weird wording considering that Merlin literally owns the county at the start
I would probably just talk about this being an important religious site, and then something about how there are many legends surrounding the Tor, including stuff about Joseph Arimathea/the Holy Thorn and the Isle of Avalon.

james9017 replied:
Wasn't sure what to write about it to be honest. I'm not well versed in the myths surrounding it. Wrote it in the frame of reference of today looking back in history. I can have a look at those.

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -141,3 +157,31 @@
+ # DUMBARTON ROCK #
+ ################## 
+ building_type_btwk_dumbarton_rock_01:0 "The Rock of Dumbarton"
+ building_type_btwk_dumbarton_rock_01_desc:0 "#italic Dumbarton Rock#! is a rocky outcrop sitting on the edge of the River Clyde overlooking the settlement of Dumbarton, providing ample defence for the fort sitting atop it. It is commonly associated with being the primary seat of the medieval Kingdom of Strathclyde and is even rumoured to have hosted the legendary wizard Merlin at one time..."
```
I would lean into the defensive aspect much more - Dumbarton Rock was said to be one of the strongest fortresses in all of Britannia (which was probably a big reason why Strathclyde survived so long, they just couldn't really fully conquer it)
I would use Alt Clut instead of Strathclyde as that's how the kingdom is localized ingame
Also not sure about "hosted the legendary wizard Merlin" - besides the fact that the tradition isn't really that relevant to why Dumbarton Rock is important, Merlin is not exactly a legendary wizard when he is a playable character in the mod...

james9017 replied:
Same issues as with drafting up Glastonbury Tor localisation. Wasn't sure what was the most appropriate thing. 

As for the use of Strathclyde, I would've put Alt Clut other than I thought we were going for the English versions of names.

DC123456789 replied:
Hmm, I guess there's a couple of things here:
- In the literature there's an increasing tendency to use Alt Clut to describe the kingdom pre-870 and Ystrad Clud/Strathclyde post-870 (when Dumbarton was sacked and the kingdom seems to have reorganized itself around Govan), so you could say that Alt Clut *is* the English name for most of the time period
- More importantly, it's confusing for players who probably won't make the connection that Alt Clut and Strathclyde are the same thing (though for this you could say something like "Kingdom of Strathclyde, also known as Alt Clut" and with a link to the title or something)

james9017 committed:
Special Building Changes - Draft 3

- Added more appropriate icons to Rock of Dumbarton & Roman Baths
- Changed building modifiers to make special buildings more powerful. Please review these to make sure they're roughly better
- Removed game start on action to add special building slots, and moved to history files correctly
- Modified localisation of: tier 4 mines desc, salt field mine type, lead/gold/iron mine type desc, most of aquae sulis baths, rock of dumbarton desc
- Still have unfinished bits in [...]. Open for suggestions. Possibly spelling mistakes, but new descs not final.
- Removed Glastonbury Tor desc for now while I think about how to approach it better.
- Yet to research salt fields/mining for tier descriptions, & yet to think of silver mine type loc.
- Removed added cultural trigger

james9017 commented:
For reference, I tried to balance the Baths (at least) against tier 2/tier 3 duchy buildings

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -34,11 +34,11 @@
  # For each, research how each was mined (doesn't matter if duplicate entries if they were acquired the same way; salt likely to be unique)
  building_type_btwk_salt_mines_01:0 "Salt Fields"
- building_type_btwk_salt_mines_01_desc:0 "Salt was a precious commodity throughout history, for its ability to preserve perishable foods like meat."
+ building_type_btwk_salt_mines_01_desc:0 "Salt's ability to preserve perishable foods - particularly meat - makes it an incredibly valuable resource in a way unique against the typically mined for metals. [something about presence in province/terrain type & how it's farmed/mined]"
```
IIRC the salt production in our area seemd to mostly be from brine pools

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -161,27 +161,27 @@
- building_type_btwk_aquae_sulis_baths_01:0 "Roman Baths"
- building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans built a series of baths around some natural springs from which the local Britons worshipped their God #italic Sulis #!(hence, the name of the settlment #italic Aquae Sulis#!). Over the following centuries during Roman occupation, this settlement would grow into a large resort complex, of which its ba ...
+ building_type_btwk_aquae_sulis_baths_01:0 "Baths of Aquae Sulis"
+ building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans occupied and developed a British shrine into a resort. The shrine (dedicated to the god #italic Sulis#!) had been built around a series of natural springs, later influencing the area's Latin name - #italic Aquae Sulis#!. These springs would later provide the foundations for the magnificent bathing co ...
```
later influencing the area's Latin name -> giving the area's Latin name
magnificent bathing complex -> magnificent baths

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -161,27 +161,27 @@
- building_btwk_aquae_sulis_baths_01:0 "Roman Baths - Restored"
- building_btwk_aquae_sulis_baths_02_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nRenovations have been carried out on the spa resort to allow for the recreational bathing the Romans once enjoyed. This has encouraged a larger influx of freemen to bring their custom to the resort - a great boon to the local economy."
+ building_btwk_aquae_sulis_baths_01:0 "Abandoned Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nUnfortunately, the departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms. The continued maintenance of amenities, like this spa, was very much an afterthought. As such, it has since fallen into disuse and decay. Many locals now instead opt f ...
```
Unfortunately, the departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms. -> Unfortunately, the baths were largely abandoned with the end of Roman Britain as the locals no longer had resources to spare on their upkeep.
The continued maintenance of amenities, like this spa, was very much an afterthought. As such, it has since fallen into disuse and decay. -> As such, like much of the other Roman-era amenities, it has largely fallen into disuse and decay.
Many locals now instead opt for bathing in the natural springs themselves. -> Only the natural springs themselves continue to be used.

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -161,27 +161,27 @@
+ building_btwk_aquae_sulis_baths_01:0 "Abandoned Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nUnfortunately, the departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms. The continued maintenance of amenities, like this spa, was very much an afterthought. As such, it has since fallen into disuse and decay. Many locals now instead opt f ...
+ building_btwk_aquae_sulis_baths_02:0 "Restored Baths"
+ building_btwk_aquae_sulis_baths_02_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nLeft to ruin since the Empire's departure, the bathing complex has since been completely renovated. Men of all manners of life now gather in the communal hub, like citizens of the Rome of old, with both locals and travellers from further afield imparting their custom to the province - a great boon to it ...
```
the bathing complex has since been completely renovated -> the baths have now been completely renovated and restored to their former glory.
Men of all manners of life now gather in the communal hub, like citizens of the Rome of old, with both locals and travellers from further afield imparting their custom to the province - a great boon to its economy to be sure -> The baths now serve as a cultural centre and communal hub for the region where men from all manners of life can gather, like the Roman citizens of old. (Not really sure about the last part you had but you can keep it if you want)

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -161,27 +161,27 @@
- building_type_btwk_dumbarton_rock_01_desc:0 "#italic Dumbarton Rock#! is a rocky outcrop sitting on the edge of the River Clyde overlooking the settlement of Dumbarton, providing ample defence for the fort sitting atop it. It is commonly associated with being the primary seat of the medieval Kingdom of Strathclyde and is even rumoured to have hosted the legendary wizard Merlin at one time..."
- building_btwk_dumbarton_rock_01:0 "$building_type_btwk_glastonbury_tor_01$"
- building_btwk_dumbarton_rock_01_desc:0 "$building_type_btwk_glastonbury_tor_01_desc$"
+ building_type_btwk_dumbarton_rock_01_desc:0 "#italic Dumbarton Rock#! is a volcanic plug sitting on the edge of the River Clyde. It overlooks the settlement of Dumbarton and has for centuries provided ample fortification for the settlement nestled within it. So much so in fact, that many attempts to subjugate its people over the past centuries have been totally defeated, thanks to the sheer clif ...
```
"#italic Dumbarton Rock#! is a volcanic plug sitting on the edge of the River Clyde. -> On the volcanic plug of "#italic Dumbarton Rock#! sits a great fortress, the traditional seat of the Kingdom of Strathclyde, or Alt Clut. 
It overlooks the settlement of Dumbarton and has for centuries provided ample fortification for the settlement nestled within it. So much so in fact, that many attempts to subjugate its people over the past centuries have been totally defeated, thanks to the sheer cliffs surrounding most of its perimeter. -> Overlooking the River Clyde, the fortress is largely surrounded by sheer cliffs and is said to be one of the strongest and most impregnable in all of Britannia. Many have tried and failed to take the fortress over the centuries, making it a powerful stronghold for control of the region. (not sure about the second half of the last sentence, feels a bit uninspired)

DC123456789 commented on Britannia/localization/english/BTWK_buildings_l_english.yml:
```diff
@@ -49,7 +49,7 @@
 
  building_type_btwk_lead_mines_01:0 "Lead Mines"
- building_type_btwk_lead_mines_01_desc:0 "Lead was a crucial metal to sustain irrigation infrastructure (despite its toxicity) during this period, as its malleability allowed for easy manipulation into constructs like pipes."
+ building_type_btwk_lead_mines_01_desc:0 "Lead's malleability allows for the easy manipulation into constructs like pipes, and was thus essential for maintaining Rome's irrigation infrastructure. While not as important now [try and tie in WRE collapse & the loss of skilled artisans], it still has its use in various construction works."
```
While not as important now -> While the collapse of the empire and much of the Roman-era economy has significantly reduced demand for the metal, ?

DC123456789 commented on Britannia/common/buildings/BTWK_special_buildings.txt:
```diff
@@ -266,26 +265,29 @@ btwk_aquae_sulis_baths_01 = {  # Disrepair
-		monthly_prestige = 0.5
-		stress_loss_mult = 0.1
+		monthly_prestige_gain_mult = 0.15
+		stress_loss_mult = 0.25
```
Might be a bit strong?

james9017 replied:
Here's the level 3 leisure palaces which I partly based the changes on:

```

character_modifier = {
		monthly_prestige_gain_mult = 0.15
		stress_loss_mult = 0.35
		owned_hostile_scheme_success_chance_add = 25
		owned_personal_scheme_success_chance_add = 25
	}
	
	duchy_capital_county_modifier = {
		monthly_county_control_change_factor = 0.6
		monthly_county_control_change_add = 0.4
	}	
```
It already has lower stress loss mult, but I reckon I could probably reduce both by 5%?

DC123456789 replied:
Oh wow I didn't realize that vanilla had such high values... I guess 25% is probably fine then, up to you

DC123456789 commented on Britannia/common/buildings/BTWK_special_mines.txt:
```diff
@@ -564,9 +564,8 @@ btwk_iron_mines_01 = {
-        heavy_cavalry_maintenance_cost_mult = -0.02
-        pikemen_maintenance_cost_mult = -0.02
+        army_maintenance_mult = -0.01
+        men_at_arms_recruitment_cost = -0.05
```
I think the modifiers are good but the mines could probably give a bigger income bonus, I think

james9017 replied:
All mines in general you reckon, or specific ones? I did scale the income down a quite a bit across the board from vanilla mines, seeing as they were more frequent.

DC123456789 replied:
Sort of thinking in general - in vanilla even the smallest mines gets 2/3.5/5/6.5, which is several times more than even the gold mine. TBH I think the flat income could easily be doubled, even taking to account that they're somewhat more common than in vanilla.

DC123456789 committed:
Merge branch 'master' of https://github.com/DC123456789/Britannia-The-Winter-King-CK3 into Buildings

DC123456789 committed:
- Finished building localisations
- Moved around a few mines
- Adjusted Dumbarton Rock modifiers

DC123456789 committed:
- Significantly increased gold income from mines
- Restoring Hadrian's Wall now requires a decision to be taken first
- Restoring Hadrian's Wall can now be done even if it was previously torn down, though it requires one more step for each building

DC123456789 committed:
- Added monument expedition descriptions for new buildings
- Blocked most mines from appearing as targets of monument expeditions

DC123456789 committed:
- Removed erroneously added lead mine in Sanquhar
